source/assets/stylesheets/_accessibility.scss

```diff
@@ -91,12 +91,19 @@ a:focus {
 input:focus,
 textarea:focus,
 select:focus,
-button:focus,
-#global-header input:focus {
+button:focus {
   outline: 3px solid $focus-colour;
+  outline-offset: 0;
 }
 
 #global-header {
+
+  input[type=search]:focus {
+    outline: 3px solid $focus-colour;
+    /* Focus appears inside the input */
+    outline-offset: -3px;
```

Comment:
Making this `-3px` exposes a white line to the right. It’s technically correct, but given that this would require the refactoring of where search boxes live and how they’re styled across multiple services can we make it `-2px` for this PR? That normalises it across browsers in my (quick) testing. If you want to leave a comment do describing why that’s fine.